app/views/pages/home.html.erb

```diff
@@ -19,11 +19,13 @@
           Log In
         </a>
       </div>
-      <div class="mt-6">
-        <a href='/auth/keycloak' class="inline-flex items-center px-4 py-2 border border-transparent text-sm font-medium rounded-md text-black text-opacity-75 bg-white bg-opacity-75 sm:bg-opacity-25 sm:hover:bg-opacity-50">
-          Sign In Keycloak
-        </a>
-      </div>
+      <% if @keycloak_enabled %>
```

Comment:
Would this be better fitted for [Flipper](https://github.com/department-of-veterans-affairs/platform-console-api/pull/15)?

Reply:
Never used Flipper before but I like the concept. I made the changes.